README.md

```diff
@@ -2,6 +2,36 @@
 
 Disclaimer: This README has some notes, but it's not fully documented yet.
 
+## Env variables
+
+We have all the configuration for beta and prod inside `./env/index.js`, there we load 2 files, the production or the development.
```

Comment:
I have used `.env` before but never as a folder. I typically do it as `.env.development` and `.env.production`. Not saying this is wrong, just a different way. I using folders to contain similar concerns.

Reply:
Ops. Miss this message. THat's an interesting point. I followed this pattern from a book I read about Node and for me it looked like a nice way to be explicit about prod and dev, and provide default values.

[In heroku they propose to use `.env` files](https://devcenter.heroku.com/articles/node-best-practices#be-environmentally-aware)

I'm happy to revisit the way we handle env variables for this project. [So I've just created a ticket for it so we can revisit it later on.](https://github.com/ferreiro/website/issues/45)

Reply:
That is a great plan.